File: Python/bee_1036.py
from math import sqrt as raiz
a, b, c = map(float, input().split())
delta = (b ** 2) - (4 * a * c)
if a == 0:
    print('Impossivel calcular')
elif delta >= 0:
    r1 = ((b * (-1)) + raiz(delta)) / (2 * a)
    r2 = ((b * (-1)) - raiz(delta)) / (2 * a)
    print(f'R1 = {r1:.5f}\nR2 = {r2:.5f}')
else:
    print('Impossivel calcular')

File: C/bee_1036.c
#include <stdlib.h>
#include <locale.h>
#include <math.h>
int main()
{
  double a, b, c, delta, r1, r2;

  scanf("%lf %lf %lf", &a, &b, &c);
  delta = ((b * b) - (4 * a * c));

  if (a == 0){
    printf("Impossivel calcular\n");
  }
  else if (delta >= 0){
    r1 = ((b * (-1)) + sqrt(delta))/ (2.0 * a);
    r2 = ((b * (-1)) - sqrt(delta)) / (2.0 * a);
    printf("R1 = %.5lf\n", r1);
    printf("R2 = %.5lf\n", r2);
  }
  else{
    printf("Impossivel calcular\n");
  }




  return(0);
}
